### PR TITLE
Replaced Cassandra-uuid with uuid

### PR DESCRIPTION
**Pull request by hknustwmf**
Bug: T223690

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
   "homepage": "https://github.com/wikimedia/restbase-mod-table-spec",
   "dependencies": {
     "routeswitch": "~0.6.3",
-    "cassandra-uuid": "^0.1.0",
```
Comment (Pchelolo) [inline package.json]: Em.. We kinda need to depend on `uuid` to use it..

File: test/utils/test_utils.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 var assert = require('assert');
-var TimeUuid = require('cassandra-uuid').TimeUuid;
+var TimeUuid = require('uuid').TimeUuid;
```
Comment (Pchelolo) [inline test/utils/test_utils.js]: The API of the uuid package is different.

Comment (Pchelolo) [reply test/utils/test_utils.js]: https://www.npmjs.com/package/uuid

File: test/utils/test_utils.js [hunk 3]
```diff
@@ -19,10 +19,8 @@ testUtils.roundDecimal = function (item) {
     return Math.round( item * 100) / 100;
 };
 
-testUtils.testTidFromDate = function testTidFromDate(date, useCassTicks) {
-    var tidNode = new Buffer([0x01, 0x23, 0x45, 0x67, 0x89, 0xab]);
-    var tidClock = new Buffer([0x12, 0x34]);
-    return new TimeUuid(date, useCassTicks ? null : 0, tidNode, tidClock).toString();
+testUtils.testTidFromDate = function testTidFromDate(date) {
+    return new uuid({ msecs: date.getTime() }).toString();
```
Comment (Pchelolo) [inline test/utils/test_utils.js]: This is not entirely correct. As you can see, in the previous version we've used dummy but fixes clockSeq, node id, and dummy nanoseconds. This was done to make TID predictable based on the time and time only. Now it's not and some of the tests will fail.

**Comment by Pchelolo** [comment]
This module contains the common tests for restbase-mod-table-cassandra and restbase-mod-table-sqlite. To properly test your change you need to replace this module in installed node_modules directories of those packages and run `npm test` on those ones.

**Comment by Pchelolo** [comment]
Patch version bump is needed.

File: test/utils/test_utils.js (hunk 2 above)
Comment (hknustwmf) [reply test/utils/test_utils.js]: Replaced and corrected

File: test/utils/test_utils.js (hunk 3 above)
Comment (hknustwmf) [reply test/utils/test_utils.js]: Added the values into the options object. In your previous wrapper you had a counter for ns. Is that used? Do I need to write a similar wrapper for uuid?

File: package.json (hunk 1 above)
Comment (hknustwmf) [reply package.json]: I run a check now to avoid in the future

**Comment by hknustwmf** [comment]
> Patch version bump is needed.

I bumped it to 1.1.3

**Comment by Pchelolo** [comment]
The `useCassTicks` was actually never `true`, so it's all good.

**Comment by d00rman** [comment]
@Pchelolo @hknustwmf this version hasn't been published nor tagged. Let's please do that

**Comment by Pchelolo** [comment]
Done @d00rman

**Pull request merged**